NOTES-Windows.txt

```diff
@@ -17,9 +19,8 @@
  for building (using GNU/Unix shell, compiler, and tools) and at run time.
  For this option you can use Cygwin.
 
-
  Visual C++ native builds, aka VC-*
- =====================================
+ -------------------------------------
```

Comment:
Argh...  I'm sorry, please undo the `=` to `-` change.  It makes things less readable, see "Requirement details" just a few lines down.  That was uninformed advice on my part, sorry.

Reply:
rebased to remove the commit.  no prob.

Reply:
You do need to push -f...

Reply:
:)